cuda/solver/common_trs_kernels.cuh

```diff
@@ -94,8 +96,12 @@ struct CudaSolveStruct : gko::solver::SolveStruct {
         : handle{exec->get_cusparse_handle()},
           spsm_descr{},
           descr_a{},
+          num_rhs{num_rhs},
```

Comment:
does hip also need that?

Reply:
from the interface, it does, but I haven't seen it matter in practice yet. Still, we should probably keep it in here

Reply:
Scratch that - the HIP implementation only uses the single-rhs version, so it doesn't matter here